### PR TITLE
Add new layer for bav.schienennetz

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/fap_schienennetz

**Comment by pfanguin** [comment]
@mariokeusen Could you test please ?

**Comment by rfrancel** [comment]
jenkins test

**Comment by pfanguin** [comment]
see sphinx config : https://github.com/geoadmin/service-sphinxsearch/pull/270

**Comment by mariokeusen** [comment]
Checked tooltips. Looks all good. Ready for prod.

File: chsdi/templates/htmlpopup/schienennetz_point.mako [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.xtf_id')}</td>                                            
+	<td>${c['featureId'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.numero')}</td>
+    <td>${c['attributes']['numero'] or '-'}</td></tr>    
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.nom_point')}</td>
+    <td>${c['attributes']['nom_point'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.abreviation')}</td>
+    <td>${c['attributes']['abreviation'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bav.schienennetz.respdonneesabreviation')}</td>
+% if abreviation == 'SBBCFFFFS':
```
Comment (gjn) [inline chsdi/templates/htmlpopup/schienennetz_point.mako]: Such things could be solved during data import, no? Or are there other interfaces that need `SBBCFFFFS` as a string?

Comment (gjn) [inline chsdi/templates/htmlpopup/schienennetz_point.mako]: @AFoletti @ltclm Thoughts?

**Comment by gjn** [comment]
Feature search doesn't work...

**Comment by ltclm** [comment]
https://github.com/geoadmin/service-sphinxsearch/pull/270 has to be merged first

**Comment by gjn** [comment]
Look my comment there...

File: chsdi/templates/htmlpopup/schienennetz_point.mako (hunk 1 above)
Comment (pfanguin) [inline chsdi/templates/htmlpopup/schienennetz_point.mako]: Yes it's possible. I do that.

**Comment by loicgasser** [comment]
feature search is ok

**Comment by loicgasser** [comment]
jenkins test

**Comment by loicgasser** [comment]
@gjn please merge once tests pass

**Pull request merged**